opentracing-noop/pom.xml

```diff
@@ -29,7 +29,7 @@
 
     <properties>
         <main.basedir>${project.basedir}/..</main.basedir>
-        <main.java.version>1.6</main.java.version>
+        <main.java.version>1.7</main.java.version>
```

Comment:
What about no-op actually requires java 7, or is this a forgotten remnant from the `AutoCloseable` vs. `Closeable` change?